POLICIES.md

```diff
@@ -0,0 +1,87 @@
+# Zerocopy's Policies
+
+## Soundness
+
+Zerocopy is expressly designed for use in security-critical contexts. It is used
+in hardware security firmware, cryptographic implementations, hypervisors, and
+more. We understand that software in these contexts has a very high bar for
+correctness, and we take our responsibility to meet that bar very seriously.
```

Comment:
```suggestion
correctness, and we take our responsibility to exceed that bar very seriously.
```

Reply:
Discussed offline; going to leave as-is.